docs/sdk-and-tools/sdk-dapp/internal-processes/guardians.md

```diff
@@ -44,16 +44,27 @@ All signing providers (except Ledger) take care internally of formatting the tra
 After formatting the transaction and applying the signature provided by the Ledger device, the transaction must be signed by the Guardian. This is done by sending the transaction (or transactions array) to the web wallet 2FA hook. The web wallet will then prompt the user to sign the transaction with the Guardian account, and respond with the signed transaction.
 
 ```js
-import { WalletProvider } from '@multiversx/sdk-web-wallet-provider';
-
-const walletProvider = new WalletProvider('https://wallet.multiversx.com/dapp/init');
-walletProvider.guardTransactions(transactions, {
-    callbackUrl: encodeURIComponent('https://my-dapp.com'),
-});
+import { CrossWindowProvider } from "@multiversx/sdk-web-wallet-cross-window-provider";
+
+// instantiate wallet cross-window provider
+await CrossWindowProvider.getInstance().init();
+const crossWindowProvider = CrossWindowProvider.getInstance();
+crossWindowProvider.setWalletUrl(WALLET_PROVIDER_URL);
+
+// set sender
+const ledgerSenderBech32 = await this.hwProvider.getAddress();
+const ledgerSender = new Address(senderBech32);
+const sender = ledgerSender; // or "erd1...abc" witohut awaiting `getAddress()`
+crossWindowProvider.setAddress(sender);
+
+// the user signs transactions on ledger so we need to perform an extra
```

Comment:
Comma after "the user signs transactions on ledger".

Reply:
_To complete the transaction, the user will sign using their Ledger device. This requires an additional step: a confirmation popup will appear, prompting the user to approve the action, after which a new tab will open in the browser._

(or something similar)